### ISSUE TITLE
Question regarding condition entropy regularization in MMDA

### ISSUE THREAD

**Issue by jaeho3690** [emadeldeen24/AdaTime#10]
Thank you for sharing your code! 

I have a question regarding the implementation of the baseline "MMDA"

https://github.com/emadeldeen24/AdaTime/blob/2d9be6bd0542e150518afaf8aa0c6ea0823b4a3d/algorithms/algorithms.py#L264C4-L264C52

**Issue**
The code applies conditional cross-entropy regularization to the target feature representation, not the logit layer.

**Questions**

1. Is this intentional?
2. If yes, why use feature representation instead of logits?
3. Should this be changed to use the logit layer?

**Comment by emadeldeen24** [comment]
We found it to be more efficient for time series data in this way. You may change it if you found it better for your scenario.
